src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.test.tsx

```diff
@@ -107,4 +156,40 @@ describe('<TranscriptsListItemInfo /', () => {
     userEvent.click(proteinLink);
     expect(defaultProps.onProteinLinkClick).toHaveBeenCalled();
   });
+
+  it('displays metadata when it is available', () => {
+    const { queryByText } = renderComponent({
+      transcript: createGencodeBasicTranscript(),
+      expandMoreInfo: true
+    });
+    const metadataLabel = queryByText('gencode basic');
+    expect(metadataLabel).toBeTruthy();
+  });
+
+  it('displays CCDS when it is available in external references', () => {
+    const { queryByText } = renderComponent({
+      transcript: createCCDSXrefTranscript(),
+      expandMoreInfo: true
+    });
+    const CCDSLabel = queryByText('CCDS');
+    expect(CCDSLabel).toBeTruthy();
+  });
+
+  it('displays Refseq when ncbi_transcript is available for MANE metadata', () => {
```

Comment:
`displays Refseq when ncbi_transcript is available for MANE metadata` -> just `displays the RefSeq` (we've established that ncbi_transcript should always be available for MANE metadata)